### ISSUE TITLE
Sideloading Issue with OfficeJS Outlook Add-in on iOS

### ISSUE THREAD

**Issue by TheSamsterZA** [OfficeDev/office-js-docs#1415]
## Article URL

https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-in-availability#outlook

## Issue

According to the above article, **Mailbox 1.5** is supported in **Outlook for iOS**. However, I cannot get my side-loaded add-in to appear on my iOS device.

[I have chronicled my adventure with this on StackOverflow here,](https://stackoverflow.com/questions/49789438/sideloading-issue-with-officejs-outlook-add-in-on-ios) and have yet to receive a proper response.

Is the documentation incorrect? Is there something I am missing? Please help.

**Comment by TheSamsterZA** [comment]
It is worth noting that my side-loaded add-in appears on Outlook for Android, and I am able to test as I expected. So this is definitely an issue specifically on the iOS side.

**Comment by mlafleur** [comment]
Additional details/information can be found in the `office-js` report in [Issue #124](https://github.com/OfficeDev/office-js/issues/124#issuecomment-388665983)

**Comment by TheSamsterZA** [comment]
Hello,

I had commented on https://github.com/OfficeDev/office-js/issues/124 but it has been closed, so perhaps no-one was notified.

**TL;DR: Please double-check with the Office for iOS client team about Mailbox 1.5 support.** It appears that this might have been working before; see https://github.com/OfficeDev/office-js-docs/issues/1301#issue-293182064.

For reference, the two places that the documentation was updated are:

1. [Add req set 1.5 support to Outlook for iPhone](https://github.com/OfficeDev/office-js-docs/commit/42d3a02f1a5ff43e2ac8f8110260d88b24b0b2ec#diff-4b8066da9eb5724afeeb500551c29b16) - also references aforementioned https://github.com/OfficeDev/office-js-docs/issues/1301.
**Published doc:** [Understanding Outlook API requirement sets](https://dev.office.com/reference/add-ins/outlook/tutorial-api-requirement-sets?product=outlook&version=v1.5).
2. [Add Outlook for iOS support for req set 1.5](https://github.com/OfficeDev/office-js-docs-pr/commit/2e8605852bea817c8b4a138e43827c8b0d59445a#diff-f950220bc173ee2422366001e5703dc2).
**Published doc:** [Office Add-in host and platform availability#outlook](https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-in-availability#outlook).

**Comment by mlafleur** [comment]
I did see your comment but thank you for posting it here as well. My purpose in consolidating these was simply to make it easier to track the issue. I suspect this is a documentation error but I will leave that determination to @jasonjoh. He's far more knowledge and equipped to determine where the root cause is than I am.

**Comment by TheSamsterZA** [comment]
As of 4 July 2018, `Outlook for iOS 2.83.0` now correctly side-loads my add-in for testing.
